daemon/start_unix.go

```diff
@@ -3,70 +3,20 @@
 package daemon // import "github.com/docker/docker/daemon"
 
 import (
-	"fmt"
-	"os/exec"
-	"path/filepath"
-
-	"github.com/containerd/containerd/runtime/linux/runctypes"
-	v2runcoptions "github.com/containerd/containerd/runtime/v2/runc/options"
 	"github.com/docker/docker/container"
-	"github.com/docker/docker/errdefs"
-	"github.com/pkg/errors"
 )
 
-func (daemon *Daemon) getRuntimeScript(container *container.Container) (string, error) {
-	name := container.HostConfig.Runtime
-	rt := daemon.configStore.GetRuntime(name)
-	if rt == nil {
-		return "", errdefs.InvalidParameter(errors.Errorf("no such runtime '%s'", name))
-	}
-
-	if len(rt.Args) > 0 {
```

Comment:
Note, I did not carry this over since I have no idea the purpose of it and it doesn't seem to be documented.

We can add it back, but I figured since it seems strange to begin with and probably no one knows the reason for it (if you do, let's document it better), just remove it. If it causes a regression we can fix it.

Reply:
Yay I found the reason for this.